### PR TITLE
Greg profile mvp

### PR DESCRIPTION
**Pull request by Gfranklin20**
This pull request also has a slight change to the chat portion so that users can now enter a bunch of more tags in the chat box

### CODE REVIEW

File: src/main/java/codeu/controller/ProfileServlet.java [hunk 1]
```diff
@@ -94,6 +95,8 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
               response.sendRedirect("/login");
               return;
             }
+
```
Comment (AGawrys) [inline src/main/java/codeu/controller/ProfileServlet.java]: Remember to keep spacing consistent.

**Review by bandreasv — COMMENTED** [review]
Overall this pull requests is good. My only big concern is that this is too big of a pull request. Try to keep your pull request smaller.

File: src/main/java/codeu/controller/ProfileServlet.java [hunk 2]
```diff
@@ -70,6 +84,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
                 return;
   }
               request.setAttribute("userProfile", userProfile);
+              //request.setAttribute("AboutMe", about);
```
Comment (AGawrys) [inline src/main/java/codeu/controller/ProfileServlet.java]: Make sure to remove any unnecessary commented out code.

**Review by AGawrys — CHANGES_REQUESTED** [review]
Good amount of code for a pull request, and it is clean. Nice! If you're having trouble with storing the About me data, have you tried maybe making it an attribute of a user? So that data could be stored when a user is created and then simply updated or changed? Not sure. Maybe fix that issue first before merging the branch into master.

**Pull request merged**